spark/tox.ini

```
@@ -25,4 +26,5 @@ commands =
     pip install -r requirements.in
     pytest -v {posargs}
 setenv =
-    2.4: SPARK_VERSION=2.4.0
+    2.4: SPARK_VERSION=2.4.0-hadoop2.7
+    3.0.1: SPARK_VERSION=3.0.1-hadoop3.2
```

```suggestion
    2.4: SPARK_VERSION=2.4.0
    2.4: SPARK_IMAGE=2.4.0-hadoop2.7
    3.0.1: SPARK_VERSION=3.0.1
    3.0.1: SPARK_IMAGE=3.0.1-hadoop3.2
```
(nit) `2.4.0-hadoop2.7` is more the image tag